README.md

```diff
@@ -12,6 +12,9 @@ The Cloud Foundation toolkit (henceforth, CFT) includes the following parts:
   single or multiple CFT-compliant config files - see:
   - The CFT source Python files (the `src/` directory)
   - The [CFT DM User Guide](dm/docs/userguide.md)
+- A growing set of [Config
+  Connector](https://cloud.google.com/config-connector/docs/overview)
+  YAML deployments: - see [the solutions directory](config-connector/solutions/)
```

Comment:
I think you'll want to remove this change from all your dev branches. Currently, we only want it to be updated in #598.

Reply:
I have erased this change and simplified the history.